.github/workflows/maintainer_merge_review.yml

```
@@ -28,7 +28,7 @@ jobs:
           type: 'stream'
           topic: 'maintainer merge'
           content: |
-            ${{ format('{0} requested a maintainer merge from review on PR [#{1}]({2}):', github.event.review.user.login, github.event.pull_request.number, github.event.pull_request.html_url ) }}
+            ${{ format('{0} requested a maintainer merge from review (submitted: ${{ pull_request_review.submitted }} or edited: ${{ pull_request_review.edited }}) on PR [#{1}]({2}):', github.event.review.user.login, github.event.pull_request.number, github.event.pull_request.html_url ) }}
```

```suggestion
            ${{ format('{0} requested a mintainer merge from review (submitted: ${{ pull_request_review.submitted }} or edited: ${{ pull_request_review.edited }}) on PR [#{1}]({2}):', github.event.review.user.login, github.event.pull_request.number, github.event.pull_request.html_url ) }}
```
Introducing a typo seems like a good idea.